### PR TITLE
SapMachine #1544: Fix wiki action

### LINKED ISSUE #1544: Run wiki update task regularly
Update [Blogs](https://github.com/SAP/SapMachine/wiki/Blogs) wiki page regularly using a GitHub action.

### PR DESCRIPTION
**Pull request by parttimenerd**
Again.

fixes #1544

### CODE REVIEW

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

File: .github/actions/update-wiki/action.yml [hunk 1]
```diff
@@ -5,11 +5,19 @@ description: 'This action updates the wiki with the latest blog posts.'
 runs:
   using: composite
   steps:
+  - uses: actions/checkout@v3
```
Comment (RealCLanger) [inline .github/actions/update-wiki/action.yml]: Why not @v4 ?

Comment (parttimenerd) [reply .github/actions/update-wiki/action.yml]: Oh, my fault.

**Comment by RealCLanger** [comment]
I fixed the PR text that it should comply now. (Removed the `.` after `Fixes #1533`).

Do you know that you can test it, if you push your PR branch to the SapMachine repo and then trigger the run via workflow_dispatch and select your branch?

File: .github/actions/update-wiki/action.yml [hunk 2]
```diff
@@ -5,11 +5,19 @@ description: 'This action updates the wiki with the latest blog posts.'
...
+      with:
+        repository: 'git@github.com:SAP/SapMachine.wiki.git'
+        ref: 'master'
+        token:  ${{ secrets.ACTIONS_GITHUB_TOKEN }}
+    - name: setup git config
+      run: |
+        git config user.name "GitHub Actions Bot"
```
Comment (RealCLanger) [inline .github/actions/update-wiki/action.yml]: Use "SapMachine Github Actions Bot"

File: .github/actions/update-wiki/action.yml [hunk 3]
```diff
@@ -5,11 +5,19 @@ description: 'This action updates the wiki with the latest blog posts.'
...
+        git config user.email "<>"
```
Comment (RealCLanger) [inline .github/actions/update-wiki/action.yml]: And sapmachine@sap.com

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by SapMachine** [comment]
Hello @parttimenerd, I'm sorry, this pull request doesn't meet the expectations. The pull request description has an invalid format. Please have a look at https://github.com/SAP/SapMachine/wiki/Formal-Requirements-of-Pull-Requests .

**Comment by parttimenerd** [comment]
It works now in my personal repository.

File: .github/workflows/wiki.yml [hunk 4]
```diff
@@ -11,7 +11,21 @@ jobs:
   wiki:
     runs-on: ubuntu-latest
     steps:
-      - name: 'Checkout SapMachine source'
-        uses: actions/checkout@v4
-      - name: 'Update Wiki'
-        uses: ./.github/actions/update-wiki
+      - uses: actions/checkout@v4
```
Comment (RealCLanger) [inline .github/workflows/wiki.yml]: Maybe add `name: 'Checkout SapMachine Wiki source'` here?

**Comment by SapMachine** [comment]
Hello @parttimenerd, this pull request fulfills all formal requirements.

**Comment by SapMachine** [comment]
Hello @parttimenerd, this pull request fulfills all formal requirements.

**Review by MBaesken — APPROVED** [review]

**Review by RealCLanger — APPROVED** [review]

**Pull request merged**